src/main/java/nz/co/jammehcow/jenkinsdiscord/DiscordWebhook.java

```diff
@@ -168,13 +170,21 @@ DiscordWebhook setFile(InputStream is, String filename) {
      * @throws WebhookException the webhook exception
      */
     public void send() throws WebhookException {
+    	
         this.embed.put("fields", fields);
         if (this.embed.toString().length() > 6000)
             throw new WebhookException("Embed object larger than the limit (" + this.embed.toString().length() + ">6000).");
 
         this.obj.put("embeds", new JSONArray().put(this.embed));
 
         try {
+        	if( jenkins.model.Jenkins.getInstance()!=null && jenkins.model.Jenkins.getInstance().proxy != null ){
```

Comment:
`Jenkins.getInstance()` is deprecated, please use `getInstanceOrNull()`

Reply:
Thanks for the advices, I made the changes.